package.json

```diff
@@ -28,11 +28,13 @@
     "google-auth-library": "^9.4.2"
   },
   "devDependencies": {
+    "@types/sinon": "^17.0.1",
```

Comment:
Please do not use `sinon` for mocking. Use the native Node.js mocks as we do elsewhere.

Reply:
Filed an issue on the repo to migrate from sinon: https://github.com/google-github-actions/analyze-code-security-scc/issues/15